server/app.router.js

```diff
@@ -1,9 +1,62 @@
 import express from 'express';
 
 import employeeRouter from './api/employee/employee.router.js';
+import Customer from './models/customer.js';
+import Wallet from './models/wallet.js';
+import customerRouter from './api/customers/customer.router.js';
+import transactionRouter from './api/transaction/transaction.router.js';
+import walletRouter from './api/wallet/wallet.router.js';
 
+// const Customer =require('../models/customer.js');
 let router = new express.Router();
 
-router.use('/employees', employeeRouter);
+router.use('/transactions', transactionRouter);
+router.use('/wallet',walletRouter);
```

Comment:
either use plural or singular pattern for resource names in your url, don't mix both